portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,6 +38,7 @@
 public class DataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setCharacterEncoding("UTF-8");
```

Comment:
Change the string literal to a constant. If this constant is used in other files as well, you can create a new file and place all these constants.